src/components/user-info/UserInfo.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import './UserInfo.css';
+
+class UserInfo extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      user: '',
+      submitted: false
+    };
+
+    this.handleChange = this.handleChange.bind(this);
```

Comment:
Remember that you can use arrow functions too. This is an accepted practice too, tho.